cfme/common/vm_views.py

```diff
@@ -435,8 +434,7 @@ class form(BasicProvisionFormView):  # noqa
 
     @property
     def is_displayed(self):
-        # Nothing is shown
-        return False
+        return self.title.text == 'Migrate Virtual Machine'
```

Comment:
QUESTION: what is it for ? 
It isn't good idea to define is_displayed for forms

Reply:
It is to differentiate when the view is migrate or cloneor ProvisionView.  The difference is that there is not the image selction in clone/migrate. because they inheriate BasicProvisionForm also